internal/web/handlers/webhandl.go

```diff
@@ -0,0 +1,88 @@
+package webhandl
```

Comment:
```suggestion
package handlers
```
имя директории = имя пакета
лишнее и бессмысленное лучше не писать и делать названия короче